flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -93,6 +110,38 @@ def bar_chart(
     return chart_specs
 
 
+def histogram(
```

Comment:
Now we have two functions that do nothing really, and exist for an opaque reason - I believe almost only because we need their names.

They at least need a docstring.